static/src/stylesheets/module/signup/_newsletters.scss

```diff
@@ -0,0 +1,151 @@
+.newsletters-heading {
+    @include fs-header(5);
+    padding: $gs-baseline 0;
+
+    @include mq($from: tablet) {
+        font-size: 36px;
+        margin-top: 8px;
+  }
+}
+
+.newsletter-card__signup {
+    padding: .5rem;
+}
+
+.newsletter-card__example {
+    display: inline;
+    float: right;
+
+    .newsletter-card__lozenge {
+        color: $news-main-1;
+        border-color: $news-main-1;
+        -webkit-border-radius: 62.5rem;
+        border-radius: 100px;
+        height: 2rem;
+        width: 6rem;
+        margin: 0;
+
+        &:hover,
+        &:focus,
+        &:active {
+            border-color: lighten($news-main-1, 10%);
+        }
+    }
+}
+
+.newsletter-card__wrapper {
+    position: relative;
+
+    @include mq(desktop) {
+        margin-left: 17%;
+    }
+}
+
+.newsletter-card {
+    color: $neutral-1;
+    display: inline-block;
+    float: left;
+    height: 16.5rem;
+    width: 90%;
+    margin: $gs-baseline 5%;
+    background-color: $neutral-6;
+    border-top: $gs-baseline/4 solid $news-main-2;
+    border-bottom: $gs-baseline/4 solid $neutral-3;
+
+    @include mq(tablet) {
+        height: 18rem;
+        width: 20rem;
+        margin: $gs-baseline;
+    }
+
+    @include mq(desktop) {
+        height: 21rem;
+        width: 30%;
```

Comment:
I've had a go with using the facia classes but it may require more css, or some extra logic in the template: Email categories (news, comment, etc) can have one to many 'cards' within them. The `l-row l-row--cols-x` wrapper doesn't seem to account for pushing a card down to the next line to wrap them into a viewport; everything is stuck in the same 'row'.

Essentially, `l-row l-row--cols-3` works great if there are exactly three 'cards' inside the wrapper DIV, however it causes suboptimal display with more or less.